server/base/src/main/java/org/apache/accumulo/server/Accumulo.java

```diff
@@ -168,8 +168,8 @@ public static void init(VolumeManager fs, ServerConfigurationFactory serverConfi
     logConfigWatcher.start();
 
     // Makes sure the log-forwarding to the monitor is configured
-    int logPort = conf.getPort(Property.MONITOR_LOG4J_PORT);
-    System.setProperty("org.apache.accumulo.core.host.log.port", Integer.toString(logPort));
+    int logPort[] = conf.getPort(Property.MONITOR_LOG4J_PORT);
```

Comment:
Can you put brackets after the type instead of the variable?  This is form is valid but [discouraged](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/arrays.html).  This occurs a couple different times in the PR.